### PR TITLE
Task links 5 - Check before merge 

### PR DESCRIPTION
**Pull request by FilipKalicanin**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -248,4 +251,186 @@ body {
...
+    }
+}
+
+@media (max-width: 1655px) {
+    .display-data {
+        width: 310px;
+    }
+}
+
+@media (max-width: 1575px) {
```
Comment (markoprogrammer) [inline assets/css/style.css:274]: This values should be standard ones, for responsive there are standard values like 768p or like from this images.:
image
so refactor it to be standard breakpoints, on all places.

File: src/mainClass.js [hunk 2]
```diff
@@ -93,37 +97,63 @@ export class AllItems {
...
     }
-    this.updateStorageCategories(el.name, el.important);
   }
-
-  updateStorageCategories(id, value) {
-    this.categoriesStorage.setItem('categories', JSON.stringify([id, value]));
+  updateStorageCategories() {
+    localStorage.setItem('importantCategories', JSON.stringify(this.mapOfImportantCategories));
+  }
+  setMapOfImportantCategories(element) {
```
Comment (markoprogrammer) [inline src/mainClass.js:123]: setCategoryAsImportant(

File: src/displayUI.js [hunk 3]
```diff
@@ -6,52 +6,64 @@ import { instanceOfMainClass } from './mainClass';
 // Display one Category element
 export function displayOneCategory(categoryElement) {
 
+  let searchContent = document.querySelector("#searchBar");
```
Comment (markoprogrammer) [inline src/displayUI.js:9]: Wrong, not read input, you already have in class filter state, so don't read from ui if (searchContent.value === '') {
you have in model that state of input.

File: src/displayUI.js [hunk 4]
```diff
@@ -6,52 +6,64 @@ import { instanceOfMainClass } from './mainClass';
...
 
   let categoryItemSvg = document.createElement('div');
   categoryItemSvg.className = "fas fa-star";
+  categoryItemSvg.id = `${elementName}Star`;
+
+  if (searchContent.value === '') {
+    category.classList.add('fadein-animation');
+  }
+
+  if (instanceOfMainClass.selectedCategory && instanceOfMainClass.selectedCategory === categoryElement.name) {
```
Comment (markoprogrammer) [inline src/displayUI.js:27]: Tend, to use seters, to follow enccapsulation.

File: assets/css/style.css (hunk 1 above)
Comment (FilipKalicanin) [reply assets/css/style.css:274]: Yes, okay. I've just tried to use media-queries wherever change was needed and that is why I had so many of them, but okay, i can do it only for standard ones.

File: src/displayUI.js (hunk 4 above)
Comment (FilipKalicanin) [reply src/displayUI.js:27]: Here i've just checked if slectedCategoy is true and if its value is same as name of current element, i haven't change it. Why is it not following encapsulation?

File: public/assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,342 @@
...
+        width: 260px;
+        font-size: .9;
+    }
+    .search-bar {
+        width: 8rem;
+        height: 2rem;
+        margin-left: 1.1rem;
+    }
+    .search-bar::placeholder {
+        font-size: .8rem;
```
Comment (markoprogrammer) [inline public/assets/css/style.css]: You copied files in public, let's remove clones.

File: public/index.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+            <button type="button" id="btnClear" class="btn-clear">Clear</button>
+        </div>
+        <div class='categories' id='categories'></div>
+    </div>
+
+    <div class="search-section" id="chosenCategorySearch">
+        <h3 class="heading">Selected:</h3>
+        <div id="displaySelected">
+            <!-- <div class="chosen-category-div">
+                <p id='chosenCategory' class='chosen-category'>asdadassssss</p>
```
Comment (markoprogrammer) [inline public/index.html]: remove comment

File: public/index.html [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    <div class="links-section">
+        <div class="heading-categories">
+            <h2 class='heading'>Links:</h2>
+            <input type="text" id="searchBarLinks" class='search-bar' placeholder="Search Links" maxlength="30"></input>
+            <button type="button" id="btnClearLinks" class="btn-clear">Clear</button>
+        </div>
+        <div class="one-category-box" id="oneCategoryDisplayBox"></div>
+    </div>
+<script type="text/javascript" src="main.bundle.js"></script></body>
+
```
Comment (markoprogrammer) [inline public/index.html]: don't commit public/main bundle/map and everything build based, your mistake was for firebase to use public.
usually you could use dist/build for firebase and add it dist/build into git ignore.

**Pull request merged**